jetty-server/src/main/java/org/eclipse/jetty/server/HttpInput.java

```diff
@@ -139,9 +143,23 @@ public int read(byte[] b, int off, int len) throws IOException
     {
         synchronized (_inputQ)
         {
+            long now=System.currentTimeMillis();
```

Comment:
The latest changes do not depend on `Request.getTimeStamp()`, so we can use System.nanoTime() for variable `now` and for field `_firstByteTimeStamp`.